.gitignore

```diff
@@ -10,4 +10,5 @@ docs/jsdoc
 
 *.env
 *.env.local
+.vscode/settings.json
```

Comment:
@hardillb @sammachin Should this file be ignored? Feels like users should have a global ignore file that keeps editor related files.